### PR TITLE
Command line: Validate email addresses manually, and when creating users

### PR DESCRIPTION
**Pull request by grischard**
Users can be created through the command line. This automatically validates the email address for users created in this way.

This also adds the 'validate' CLI user command to manually validate an user's email address confirmation request.

### CODE REVIEW

File: udata/core/user/commands.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def create():
         data['password'] = encrypt_password(data['password'])
         del data['password_confirm']
         user = datastore.create_user(**data)
+        user.confirmed_at = datetime.utcnow()
```
Comment (davidbgk) [inline udata/core/user/commands.py]: I think that you'll have to add that property to the `User`'s model too?

**Comment by davidbgk** [comment]
The manual `password` command should set the attribute too(?)

File: udata/core/user/commands.py (hunk 1 above)
Comment (grischard) [inline udata/core/user/commands.py]: It already is: https://github.com/etalab/udata/blob/3c2aa6ace82da212b30de6012174e866c3df312c/udata/core/user/models.py#L67

Comment (davidbgk) [inline udata/core/user/commands.py]: Oh I just realized I made a typo, then redeclare the attribute, my bad :D

File: udata/core/user/commands.py [hunk 2]
```diff
@@ -46,6 +47,20 @@ def create():
 
 
 @m.command
+def activate():
+    '''Activate an existing user (validate their email confirmation)'''
+    email = prompt('Email')
```
Comment (davidbgk) [inline udata/core/user/commands.py]: Interesting that for `delete` we prompt the email and for `set_admin` and `password` we have it as a parameter. Maybe an opportunity to be consistent here? @noirbizarre ?

Comment (noirbizarre) [inline udata/core/user/commands.py]: :+1: for consistency.
Proposal: they all takes an optional first argument email (login in fact) that will be prompted if not provided

File: udata/core/user/commands.py [hunk 3]
```diff
@@ -46,6 +47,20 @@ def create():
...
+    user = User.objects(email=email).first()
+    if not user:
+        print 'Invalid user'
+        return
+    if user.confirmed_at is not None:
+        print 'User email address already corfirmed'
```
Comment (yohanboniface) [inline udata/core/user/commands.py]: Typo: "confirmed" ;)

File: udata/core/user/commands.py [hunk 4]
```diff
@@ -46,6 +47,20 @@ def create():
...
+    user.confirmed_at = datetime.utcnow()
```
Comment (yohanboniface) [inline udata/core/user/commands.py]: Missing a `save`?

**Pull request merged**